app/scripts/proactive/view/dom.js

```diff
@@ -998,7 +998,7 @@ define(
         })();
 
         $(document).ready(function () {
-
+            var copiedTask = [];
```

Comment:
There are several tasks inside copiedTask right? 'copiedTasks' might be more appropriate